topics/dev/tutorials/tool-from-scratch/tutorial.md

```diff
@@ -336,6 +336,18 @@ user interface, then translate the user's selections back into the command to be
 on writing a wrapper. Although this section will cover the full process and many of the options available to
 anyone wrapping a tool, a more complete list is available in [the Galaxy tool docs](https://docs.galaxyproject.org/en/latest/dev/schema.html).
 
+## Installing Planemo
```

Comment:
How about just reorganizing the tutorial and move the planemo install section from the back to this place?

Reply:
Makes sense. Fixed.